tropohelper/services.py

```diff
@@ -26,6 +35,50 @@ def create_firehose(stack, name, bucket_arn, buffering_seconds=300, buffering_si
             RoleARN="arn:aws:iam::347225174248:role/firehose_delivery_role")
         ))
 
+def create_stream(stack, name, shard_count):
+    return stack.stack.add_resource(Stream(
+        '{0}Stream'.format(name.replace('-', '')),
+        ShardCount=shard_count
+    ))
+
+def create_json_redshift_firehose_from_stream(stack, name, firehose_arn,
+                                              source_stream_arn, source_stream_role_arn,
+                                              redshift_host_port, redshift_db_name,
+                                              redshift_username, redshift_password,
+                                              redshift_db_table_name,
+                                              s3_bucket_arn, s3_buffering_seconds=300, s3_buffering_size=5):
+    """Add Kinesus Redshift Firehose Resource with another Kinesis Stream as source."""
+    return stack.stack.add_resource(DeliveryStream(
+        '{0}Firehose'.format(name.replace('-', '')),
+        DeliveryStreamName=name,
+        DeliveryStreamType=KinesisStreamAsSource,
+        KinesisStreamSourceConfiguration=KinesisStreamSourceConfiguration(
+            KinesisStreamARN=source_stream_arn,
+            RoleARN=source_stream_role_arn
+        ),
+        RedshiftDestinationConfiguration=RedshiftDestinationConfiguration(
+            ClusterJDBCURL='jdbc:redshift://%s/%s' % (redshift_host_port, redshift_db_name),
```

Comment:
Please use the same type of string formatting being used already in this library (.format)

Reply:
done